sql/src/main/java/io/crate/planner/consumer/ManyTableConsumer.java

```diff
@@ -129,6 +140,38 @@ public Plan consume(AnalyzedRelation relation, ConsumerContext context) {
         return bestOrder;
     }
 
+    private enum RelationInclusion {
+        BOTH_INCLUDED,
+        FOREIGN_INCLUDED,
```

Comment:
Might want to move the documentation that is currently part of `determineRelationInclusion` to each value here